src/Processor/LastResort.php

```diff
@@ -40,8 +40,10 @@ public function copy(array $state, Result $result, int $timeLimit = PHP_INT_MAX)
         $bytesCopied = 0;
         $fin = fopen($from, "rb");
         $fout = fopen($to, "w");
-        while (!feof($fin)) {
-            $bytesCopied += fwrite($fout, fread($fin, $bytesToRead));
+        if ($fin) {
+            while (!feof($fin)) {
```

Comment:
@Thierry What does feof() return on a handler of a file that did not open successfully?

Reply:
```
php > $fin = fopen("not_here.txt", "r");
php > echo var_export($fin, TRUE);
false
php > echo var_export(feof($fin), TRUE);
false
```
Hence the infinite loop, filling up with warnings like `feof() expects parameter 1 to be resource, bool given in php shell code on line ...`

Reply:
A lot of things can go wrong within (original) lines 43 and 44, so we might need to test the outcome of every single function.